dotnet6/cookiecutter-aws-sam-quickstart-sns-dotnet/{{cookiecutter.project_name}}/template.yaml

```diff
@@ -0,0 +1,51 @@
+# This is the SAM template that represents the architecture of your serverless application
+# https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-template-basics.html
+
+# The AWSTemplateFormatVersion identifies the capabilities of the template
+# https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/format-version-structure.html
+AWSTemplateFormatVersion: 2010-09-09
+Description: >-
+  {{cookiecutter.project_name}}
+
+# Transform section specifies one or more macros that AWS CloudFormation uses to process your template
+# https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/transform-section-structure.html
+Transform:
+- AWS::Serverless-2016-10-31
+
+Globals:
+  Function:
+    Timeout: 100
+
+# Resources declares the AWS resources that you want to include in the stack
+# https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/resources-section-structure.html
+Resources:
+  # This is an SNS Topic with all default configuration properties. To learn more about the available options, see
+  # https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-sns-topic.html
+  SimpleTopic:
+    Type: AWS::SNS::Topic
+  
+  # This is the Lambda function definition associated with the source code: sns-payload-logger.js. For all available properties, see
+  # https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessfunction
+  SNSPayloadLogger:
+    Type: AWS::Serverless::Function
+    Properties:
+      Description: A Lambda function that logs the payload of messages sent to an associated SNS topic.
```

Comment:
@neville-lewis can you please add these proprieties to make it ease to select Graviton (ARM64) architecture 

[https://github.com/aws/aws-sam-cli-app-templates/blob/6aaf6ac2e0ada5f87c4c6f69c5c1d8696513f2fe/dotnet6/cookiecutter-aws-sam-hello-step-functions-sample-app/%7B%7Bcookiecutter.project_name%7D%7D/template.yaml#L68](https://github.com/aws/aws-sam-cli-app-templates/blob/6aaf6ac2e0ada5f87c4c6f69c5c1d8696513f2fe/dotnet6/cookiecutter-aws-sam-hello-step-functions-sample-app/%7B%7Bcookiecutter.project_name%7D%7D/template.yaml#L68)